src/utils/pgwire/src/pg_protocol.rs

```diff
@@ -460,26 +554,12 @@ where
     async fn process_query_msg(&mut self, query_string: io::Result<&str>) -> PsqlResult<()> {
         let sql: Arc<str> =
             Arc::from(query_string.map_err(|err| PsqlError::SimpleQueryError(Box::new(err)))?);
-        let start = Instant::now();
+        record_sql_in_span(&sql);
```

Comment:
a little confused. Does this duplicate with the layer in `do_process`?

Reply:
> the layer in do_process?

What do you mean by this? 😳

Reply:
Oh, `do_process` emits the log line, but `process_query_msg` adds the sql field. I see.